packages/client/tests/functional/issues/9326/tests.ts

```diff
@@ -50,24 +51,29 @@ testMatrix.setupTestSuite(
         expect(tags.length).toBe(n)
       })
 
-      test('should fail with `Assertion violation` when the number of params is 32768+', async () => {
-        expect.assertions(3)
-        const n = 32768
-        const ids = await createTags(n)
+      // TODO this test does not fail on adapters, and has zero assertions. Are we creating prepared statements?
```

Comment:
Like previous comment
Actually, it's expected that this does not fail for driver adapters, since the failure is only expected when using a prepared statement.